oak-core-spi/src/main/java/org/apache/jackrabbit/oak/spi/mount/SimpleMountInfoProvider.java

```diff
@@ -18,47 +18,50 @@
  */
 package org.apache.jackrabbit.oak.spi.mount;
 
+import org.jetbrains.annotations.NotNull;
+
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 /**
  * A simple and inefficient implementation to manage mount points
  */
 final class SimpleMountInfoProvider implements MountInfoProvider {
 
-    private final Map<String, Mount> mounts;
+    private final Map<String, Mount> nameToMounts;
```

Comment:
For me, "nameToMounts" would mean the key is name, and the value is an _array_ of mounts... So I would use "nameToMount". 

```suggestion
    private final Map<String, Mount> nameToMount;
```

Reply:
Done